x-pack/test/functional/apps/ml/anomaly_detection/lens_to_ml.ts

```diff
@@ -0,0 +1,222 @@
+  const browser = getService('browser');
+  const retry = getService('retry');
+  const headerPage = getPageObject('header');
+  const PageObjects = getPageObjects(['common', 'timePicker', 'dashboard']);
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+
+  const dashboardTitle = 'lens_to_ml';
+
+  async function retrySwitchTab(tabIndex: number, seconds: number) {
+    await retry.tryForTime(seconds * 1000, async () => {
+      await browser.switchTab(tabIndex);
+    });
+  }
+
+  async function setFarequoteTimerange() {
+    await PageObjects.timePicker.setAbsoluteRange(
+      'Feb 7, 2016 @ 00:00:00.000',
+      'Feb 11, 2016 @ 23:59:54.000'
+    );
+  }
+
+  async function dashboardPreparation(selectedPanelTitle: string) {
+    await PageObjects.dashboard.loadSavedDashboard(dashboardTitle);
+    await ml.dashboardEmbeddables.assertDashboardPanelExists(selectedPanelTitle);
+
+    await setFarequoteTimerange();
+
+    const header = await dashboardPanelActions.getPanelHeading(selectedPanelTitle);
+    await dashboardPanelActions.openContextMenuMorePanel(header);
+  }
+
+  async function createJobInWizard(
+    jobId: string,
+    splitField?: string,
+    aggAndFieldIdentifier?: string
+  ) {
+    await headerPage.waitUntilLoadingHasFinished();
+
+    if (splitField !== undefined) {
+      await ml.jobWizardMultiMetric.assertDetectorSplitExists(splitField);
+      await ml.jobWizardCommon.assertInfluencerSelection([splitField]);
+    }
+
+    if (aggAndFieldIdentifier !== undefined) {
+      await ml.jobWizardCommon.assertAggAndFieldInputExists();
+      await ml.jobWizardCommon.selectAggAndField(aggAndFieldIdentifier, true);
+      await ml.jobWizardCommon.assertAnomalyChartExists('LINE');
+    }
+
+    await ml.testExecution.logTestStep('job creation displays the job details step');
+    await ml.jobWizardCommon.advanceToJobDetailsSection();
+
+    await ml.testExecution.logTestStep('job creation inputs the job id');
+    await ml.jobWizardCommon.assertJobIdInputExists();
+    await ml.jobWizardCommon.setJobId(jobId);
+
+    await ml.testExecution.logTestStep('job creation displays the validation step');
+    await ml.jobWizardCommon.advanceToValidationSection();
+
+    await ml.testExecution.logTestStep('job creation displays the summary step');
+    await ml.jobWizardCommon.advanceToSummarySection();
+
+    await ml.testExecution.logTestStep('job creation creates the job and finishes processing');
+    await ml.jobWizardCommon.assertCreateJobButtonExists();
+    await ml.jobWizardCommon.createJobAndWaitForCompletion();
+
+    await ml.testExecution.logTestStep('job creation displays the created job in the job list');
+    await ml.navigation.navigateToMl();
+    await ml.navigation.navigateToJobManagement();
+
+    await ml.jobTable.filterWithSearchString(jobId, 1);
+
+    await ml.jobTable.assertJobRowJobId(jobId);
+  }
+
+  describe('anomaly charts in dashboard', function () {
```

Comment:
I think the suite title should rather be something like `create job from lens`.

Reply:
ah yes, that's a copy and paste mistake

Reply:
https://github.com/elastic/kibana/pull/133899/commits/b05cf28b33b439400f16dcc749d72e7edac877f0